Essentials/src/net/ess3/api/events/teleport/TeleportAcceptEvent.java

```diff
@@ -0,0 +1,59 @@
+package net.ess3.api.events.teleport;
+
+import net.ess3.api.IUser;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerTeleportEvent;
+
+/**
+ * Event that's called when a player executes /tpaccept
+ * <p>
+ * Cancelling this event will prevent the user from teleporting.
+ */
+public class TeleportAcceptEvent extends Event implements Cancellable {
```

Comment:
I'd prefer this to be named `TeleportRequestAcceptEvent`

Reply:
Also, this PR doesn't account for `/tpauto` which will auto-accept requests - do we want to fire the event for this?